extensions/positron-r/src/r-installation.ts

```diff
@@ -165,23 +165,23 @@ function getRHomePathNotWindows(binPath: string): string | undefined {
 	const R_HOME_DIR = removeSurroundingQuotes(extractValue(targetLine, 'R_HOME_DIR'));
 	const homepath = R_HOME_DIR;
 	if (homepath === '') {
-		LOGGER.info(`Can\'t determine R_HOME_DIR from the binary: ${binPath}`);
+		LOGGER.info(`Can\'t determine R_HOME_DIR from the binary: ${binpath}`);
 		return undefined;
 	}
 	return homepath;
 }
 
-function getRHomePathWindows(binPath: string): string | undefined {
+function getRHomePathWindows(binpath: string): string | undefined {
 	// find right-most 'bin' in the path and take everything to the left of it
-	// Examples of binPaths:
-	// "C:\Program Files\R\R-4.3.2\bin\R.exe"     <-- the path produced by our binFragment() helper
-	// "C:\Program Files\R\R-4.3.2\bin\x64\R.exe" <-- but this also exists
-	const binIndex = binPath.lastIndexOf(path.sep + 'bin' + path.sep);
+	// Examples of binpaths:
+	// "C:\Program Files\R\R-4.3.2\bin\x64\R.exe" <-- we prefer this, if both are present
+	// "C:\Program Files\R\R-4.3.2\bin\R.exe"     <-- usually a shim for the path above
+	const binIndex = binpath.lastIndexOf(path.sep + 'bin' + path.sep);
```

Comment:
This comment was actually outdated and our preferences are the other way around.